pyproject.toml

```diff
@@ -5,7 +5,7 @@ description = "Collection of templates for Plone integrators to use through Cook
 readme = "README.md"
 requires-python = ">=3.10"
 dependencies = [
-    "cookieplone>=0.9.4",
+    "cookieplone>=0.9.6",
```

Comment:
@ericof This doesn't match what you put for MIN_COOKIEPLONE

Reply:
@ericof If it uses `>` then it is not the minimum, but one less than the minimum. So it's currently confusing.

Reply:
@davisagli My bad, just fixed